components/widgets/section-widget.js

```diff
@@ -4,31 +4,88 @@ import { WidgetSelector } from "@/components/widgets/widget-selector";
 import { getHeadingLevel } from "@/lib/string-utils";
 
 export const Section = ({ data }) => {
-  const primary = [];
-  const secondary = [];
-  const others = [];
-
-  data?.content?.forEach((widget, index) => {
-    const columnName = widget?.buttonSectionColumn?.name ?? widget?.sectionColumn?.name;
-    const column = columnName ? columnName.toLowerCase() : '';
-    switch (column) {
-      case "primary":
-        primary.push(<WidgetSelector key={index} data={widget} />);
-        break;
-      case "secondary":
-        secondary.push(<WidgetSelector key={index} data={widget} />);
-        break;
-      default:
-        others.push(<WidgetSelector key={index} data={widget} />);
-        break;
+  // Sort widgets into primary, secondary and others
+  const {
+    primary: ungroupedPrimary,
+    secondary,
```

Comment:
I'm noticing a block getting placed in the primary region instead of secondary on this page: 
http://localhost:3000/lang/commerce

<img width="1529" alt="image" src="https://github.com/user-attachments/assets/4f48e75a-4d71-47b4-95e7-3fd390d72abc">

Reply:
Can be handled via #28